python/arcticdb/version_store/library.py

```diff
@@ -426,7 +428,7 @@ def _raise_if_unsupported_type_in_write_batch(payloads):
         raise ArcticUnsupportedDataTypeException(error_message)
 
     def write_batch(
-        self, payloads: List[WritePayload], prune_previous_versions: bool = True, staged=False
+        self, payloads: List[WritePayload], prune_previous_versions: bool = True, staged=False, ensure_sorted=True
```

Comment:
Why would `write_batch` has a different default from `write`??

Reply:
I think `write` is wrong - it looks to me like it defaults to `False` but it should default to `True` in this API.

Reply:
yes, that should be True as well, thanks